docs/pages/jpsreport/jpsreport_method_C.md

```diff
@@ -0,0 +1,21 @@
+---
+title:  Method C
+keywords: analysis
+tags: [jpsreport, measurement]
+sidebar: jupedsim_sidebar
+folder: jpsreport
+summary: this method is the classical method for calculating the density. 
+permalink: jpsreport_method_C.html
+last_updated: Dec 20, 2019
+---
+
```

Comment:
Why the image was removed from the detailed description of method C?

Reply:
added..